public/dashboard/index.html

```diff
@@ -156,16 +156,16 @@
     <div class="resources-container w-container">
       <div class="w-layout-grid resources-grid">
         <div class="card-grid">
-          <div class="text-block">HOW TO <br>DEVELOP LOW COST <br>SENSORS</div>
-          <a href="#" class="button-2 w-button">GET STARTED</a>
+          <div class="text-block">LOW COST<br>SENSOR USER <br> MANUAL</div>
+          <a href="https://drive.google.com/drive/u/2/folders/1akHJdqMljDOngMzGjGXtx_GFW9X5v-nP" class="button-2 w-button">GET STARTED</a>
         </div>
         <div class="card-grid">
-          <div class="text-block">WIRING<br>LOW COST<br>SENSORS</div>
-          <a href="#" class="button-2 w-button">GET INSTRUCTIONS</a>
+          <div class="text-block">SENSOR TRAINING<br>ENUMERATOR<br>COURSEWARE</div>
+          <a href="https://docs.google.com/document/d/1zA0NVlbTgdvOCyL4HrzrmvgbyolIhuaDOFsIHT4hobo/edit" class="button-2 w-button">GET INSTRUCTIONS</a>
         </div>
         <div class="card-grid">
           <div class="text-block">AIR QUALITY SENSORS<br>OPEN SOURCE<br>FIRMWARE</div>
-          <a href="#" class="button-2 w-button">GITHUB REPO</a>
+          <a href="https://github.com/CodeForAfrica/sensors.AFRICA-AQ-sensors-software/tree/recent_luftdaten_master" class="button-2 w-button">GITHUB REPO</a>
```

Comment:
1. All these links should open on new tab/window,
2. Is the content publicly viewable e.g. can someone on the Internet view google doc?
3. any reason we're linking to a specific branch of the GitHub repo and not just main/master?

Reply:
@kilemensi The specific GitHub link shows the branch that reflects the latest changes from luftdaten repo. Also this branch is not necessarily managed by the tech team, so i can follow up on this with the sensors team and find out if we need to change that

Reply:
For the google docs I can ask constant to make them shareable for anyone with the link